src/controllers/update/update.any.trusts.involved.controller.ts

```diff
@@ -0,0 +1,38 @@
+import { NextFunction, Request, Response } from "express";
+import { logger } from "../../utils/logger";
+import * as config from "../../config";
+import { ApplicationData } from "model";
+import { getApplicationData, setExtraData } from "../../utils/application.data";
+import { AnyTrustsInvolvedKey } from "../../model/data.types.model";
+
+export const get = (req: Request, res: Response, next: NextFunction) => {
+  try {
+    logger.debugRequest(req, `${req.method} ${req.route.path}`);
+    const appData: ApplicationData = getApplicationData(req.session);
+    return res.render(config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE, {
+      backLinkUrl: config.SECURE_UPDATE_FILTER_URL,
+      templateName: config.UPDATE_ANY_TRUSTS_INVOLVED_PAGE,
+      appData,
+      [AnyTrustsInvolvedKey]: appData.any_trusts_involved
```

Comment:
No need to pass this as we're already passing appData

Reply:
Why is any_trusts_involved stored in appData? The interrupt cards don't allow you to navigate "forwards"" so does the choice need to be remembered? We don't save&continue the appData to storage so this would be something just in the Web session and not in the "update model".

Reply:
Hey Andre, yep this has already been discussed by me and Stephanie and will be updating with a fix.